dev/tests/integration/testsuite/Magento/Payment/Helper/DataTest.php

```diff
@@ -18,7 +20,62 @@ public function testGetInfoBlock()
             \Magento\Payment\Model\Info::class
         );
         $paymentInfo->setMethod('checkmo');
+        $paymentInfo->
         $result = $helper->getInfoBlock($paymentInfo);
         $this->assertInstanceOf(\Magento\OfflinePayments\Block\Info\Checkmo::class, $result);
     }
+
+    /**
+     * @magentoDataFixture Magento/Store/_files/second_store.php
+     */
+    public function testGetPaymentMethodList()
```

Comment:
There should be two tests each mentioning particular scenario.

Use annotations to set config values.

Reply:
Agree with you, done.